src/black/lines.py

```diff
@@ -49,7 +49,7 @@
 class Line:
     """Holds leaves and comments. Can be printed with `str(line)`."""
 
-    mode: Mode
+    mode: Mode = field(repr=False)
```

Comment:
This makes debugging much less verbose; plenty of other opportunities to see what the mode is.